README.md

```diff
@@ -15,11 +15,6 @@ The Outline Shadowsocks service allows for:
 
 ## Try it!
 
-Fetch auxiliary dependencies:
-```
-GO111MODULE=off go get github.com/shadowsocks/go-shadowsocks2 github.com/prometheus/prometheus/cmd/...
```

Comment:
How will the commands below work without this?

Reply:
You're right, I misunderstood the purpose of this instruction.  I've restored it.

Reply:
Do we still need `GO111MODULE=off`?

Reply:
I don't know, and I'm not sure how to test it to be sure.